db/migrations/postgres/000081_recent_searches.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS recentsearches (
```

Comment:
Just curious to know: 
How do we handle a case when there is another feature which requires DB migrations and also has the sequence `000081` and gets merged before this PR?

Reply:
The sequence number isn't really important. It's just a human readable number to distinguish. The library will see which all migrations haven't been applied and apply those. But only when you need to enforce order between multiple migrations, then first it sorts by sequence numbers, and then by the name. So if 2 people send 2 PRs, but there is a dependency between the 2 migrations, then one needs to wait for the other. The tool doesn't know this and is left at the user layer to resolve.